amy/settings.py

```diff
@@ -338,6 +338,17 @@
     },
 }
 
+# settings for PyData application
+ENABLE_PYDATA = os.environ.get('AMY_ENABLE_PYDATA', False)
+
+if ENABLE_PYDATA:
+    INSTALLED_APPS.insert(
+        INSTALLED_APPS.index('workshops'),
+        'pydata'
+    )
```

Comment:
Could you replace it with:

```python
INSTALLED_APPS = [
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    'workshops',
]
if ENABLE_PYDATA:
    INSTALLED_APPS += [
        'pydata',
    ]
INSTALLED_APPS += [
    # this should be after 'workshops' because templates in
    # 'templates/registration/' clash
    'django.contrib.admin',
    'crispy_forms',
    'selectable',
    'django_countries',
    'django_filters',
    'reversion',
    'rest_framework',
    'api',
    'extforms',
    'captcha',
    'compressor',
    'social.apps.django_app.default',
    'debug_toolbar',
]
```

So that `INSTALLED_APPS` is defined in one place.